lib/saml/post_url_service.rb

```diff
@@ -64,14 +64,22 @@ def tou_declined_logout_redirect_url
 
     def terms_of_use_redirect_url
       application = @tracker&.payload_attr(:application) || 'vaweb'
-      if TERMS_OF_USE_ENABLED_CLIENTS.include?(application)
+      if what_clients_are_enabled.include?(application)
         Rails.logger.info('Redirecting to /terms-of-use', type: :ssoe)
         add_query(terms_of_use_url, { redirect_url: login_redirect_url })
       else
         login_redirect_url
       end
     end
 
+    def what_clients_are_enabled
```

Comment:
Could you change this method name to something a bit more descriptive since there are other components in this class not related to terms of use? perhaps something like `enabled_tou_clients`. Also, should this method be `private`?

Reply:
Yes makes sense, yes it can be a private method